includes/wikia/GlobalFunctions.php

```diff
@@ -1557,3 +1557,17 @@ function wfGetBaseDomainForHost( $host ) {
 
 	return $wgWikiaBaseDomain;
 }
+
+function wfDoesURLHaveFandomDomain( $url ): bool {
+	global $wgFandomBaseDomain;
+
+	// e.g. not existing wikis
+	if ( empty( $url ) ) {
+		return false;
+	}
+
+	$host = parse_url( $url, PHP_URL_HOST );
+	$host = WikiFactory::normalizeHost( $host );
+
+	return wfGetBaseDomainForHost( $host ) === $wgFandomBaseDomain;
```

Comment:
This might not work for dev environment. For example `normalizeHost()` will return `fadnom-dev.us` then `wfGetBaseDomainForHost()` will return `wikia.com` which will be not the same as `$wgFandomBaseDomain`.

Reply:
`normalizeHost()` won't return `fandom-dev.us`, @themech fixed it in https://github.com/Wikia/app/pull/16068

Reply:
👍